packages/ethereum-wallets/src/lib/index.ts

```
@@ -528,6 +549,63 @@ const EthereumWallets: WalletBehaviourFactory<
     transactions: Array<Optional<Transaction, "signerId" | "receiverId">>
   ) => {
     const nearTxs = await transformTransactions(transactions);
+    if (bannedNearAddressesPackage === null) {
+      await importBannedNearAddressesPackage();
+    }
+
+    let restrictedActionError = "";
+    for (let i = 0; i < nearTxs.length; i++) {
+      const tx = nearTxs[i]
```

ESLint

[error] Insert `;`

(prettier/prettier)